src/components/TrackTile/TrackerDetails/to-observation-resource.ts

```diff
@@ -44,7 +44,7 @@ const toFhirObservationResource = (
     meta: {
       tag: [
         {
-          code: datastoreSettings.project,
+          code: accountSettings?.project || '',
```

Comment:
What's the idea with this change?  Assuming `to-observation-resource` is always called right before sending to the cloud, it doesn't seem like we'd want to allow for an empty string here.  We know that'd cause a 400, so seems like we should just not send the request / get to this method in those cases.

Reply:
Good catch.  This change was added last minute when I was trying to get through some typing errors. I meant to get back to it.  Didn't realize an empty string would cause an observation to fail either though... 😨